src/grammar.lalrpop

```diff
@@ -300,6 +300,17 @@ ClauseGuard3: UntypedClauseGuard = {
 }
 
 ClauseGuard4: UntypedClauseGuard = {
+    <s:@L> <left:ClauseGuard4> ">" <right:ClauseGuard5> <e:@L> => ClauseGuard::GreaterThan {
+        location: location(s, e),
+        typ: (),
+        left: Box::new(left),
+        right: Box::new(right),
+    },
+
+    ClauseGuard5 => <>,
+}
+
+ClauseGuard5: UntypedClauseGuard = {
```

Comment:
I'm really clueless about this file but followed the numbering instructions and it worked just fine :) 

Why are names numbered?

Reply:
It's to do with operator precedence. The parser tries one set of operators, then another, then another, etc. :)